app.config.ts

```diff
@@ -3,3 +3,16 @@ export const avgLearnerAttendanceLimit: number = 66;
 export const dashboardDaysLimit: number = 30;
 export const modifyAttendanceLimit: number = 6;
 export const toastAutoHideDuration: number = 5000; // 5 seconds
+
+export const dropoutReasons = [
+    { label: 'UNABLE_TO_COPE_WITH_STUDIES', value: 'unable_to_cope_with_studies' },
```

Comment:
@Rushikesh-Sonawane99 i think value can be in english... please check with backend team once.